### ISSUE TITLE
Latest release on cdnjs?

### ISSUE THREAD

**Issue by dsparacio** [google/shaka-player#3381]
Not seeing 3.0.11 or 3.1.0 on cdnjs.  When are these expected to be deployed?

**Comment by dsparacio** [comment]
https://cdnjs.com/libraries/shaka-player

**Comment by joeyparrish** [comment]
CDNjs is incredibly slow to update.  We can't control when their bot updates the CDN.

Last I saw, they stored their config and their assets in the same git repo.  The last time I had to make changes to the config, I had to check out a repo from GitHub so large that the checkout couldn't be completed.  I had to then pull new refs several times to get my local copy in sync with the remote copy, which took more than a day of checking back on the progress and pulling again.  The repo is **gigantic**.

So I assume that is why even their automated update process lags so far behind reality.

Feel free to try [Google Hosted Libraries](https://developers.google.com/speed/libraries#shaka-player), [unpkg](https://unpkg.com/shaka-player@3.1.0/dist/shaka-player.compiled.js), [jsdelivr](https://cdn.jsdelivr.net/npm/shaka-player@3.1.0/dist/shaka-player.compiled.js), on any other CDN you like.
